### PR TITLE
C21 wmde test 05

### PR DESCRIPTION
**Pull request by Abban**
Ticket: https://phabricator.wikimedia.org/T289459

### CODE REVIEW

File: desktop/styles/styles_var.pcss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+@import '../../shared/components/ui/use_of_funds/FundsModal.pcss';
+@import '../../shared/components/ui/use_of_funds/FundsContent.pcss';
+@import '../../shared/components/ui/use_of_funds/FundsDistributionInfo.pcss';
+@import '../../shared/components/ui/use_of_funds/CompanyBudgets.pcss';
+
+/* TODO: Remove this */
+#mw-notification-area {
+	display: none !important;
+}
```
Comment (moiikana) [inline desktop/styles/styles_var.pcss]: TODO left

**Comment by moiikana** [comment]
Bildschirmfoto vom 2021-09-06 15-55-51
saw some overlapping label at ~ width 1590px (firefox, linux)

File: desktop/styles/styles_var.pcss (hunk 1 above)
Comment (Abban) [reply desktop/styles/styles_var.pcss]: Whoops!

File: desktop/banner_var.js [hunk 2]
```diff
@@ -40,11 +38,10 @@ bannerPresenter.present(
...
 		formatters,
 		useOfFundsText,
-		donationForm: DonationForm,
 		footer: Footer,
-		bannerText: BannerText,
+		donationForm: DonationForm,
 		translations: Object.assign( Translations, LocalTranslations ),
 		formItems: createFormItems( Translations, formatters.amountInputFormatter ),
-		bannerType: BannerType.VAR
+		bannerType: BannerType.CTRL
```
Comment (moiikana) [inline desktop/banner_var.js:45]: not `bannerType: BannerType.VAR`?
(does not seem to do any difference here ..)

**Review by moiikana — APPROVED** [review]

**Pull request merged**